charts/tonic/values.yaml

```diff
@@ -74,10 +74,10 @@ tonicai:
       http: 80
     resources:
       requests:
-        memory: "2Gi"
+        memory: "4Gi"
         ephemeral-storage: "1Gi"
       limits:
-        memory: "3Gi"
+        memory: "5Gi"
```

Comment:
Don't we want the limit and the request to be equal?  I thought that was best practice.

Reply:
Updated